src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/getPaymentMethod/payment.js

```diff
@@ -33,7 +33,7 @@ function handlePaymentMethod({ req, res, next }) {
     ImagePath: adyenURL,
     AdyenDescriptions: paymentMethodDescriptions,
     AdyenConnectedTerminals: JSON.parse(connectedTerminals),
-    amount: { value: paymentAmount, currency },
+    amount: { value: parseFloat(paymentAmount), currency },
```

Comment:
Where is this amount used? In case it's is used for the frontend components as well as a config field you need to use minor units.

Reply:
@cyattilakiss yes it is used in the frontend. But I tried it in google pay and it works fine. Maybe minor units are required starting from a newer version?

Reply:
it is required from the beginning. It won't always break the implementation but for example Apple pay would be immediately broken because of it and it can also cause problems in other components that we are not aware at the moment :(